components/Typography/typography.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/main.css">
+    <title>Calm_UI | Typography</title>
+</head>
+
+<body>
+    <div class="container">
+        <header class="head-nav">
+            <h2><a href="/index.html" class="logo">Calm UI</a></h2>
+            <div class="navigation">
+                <a href="/pages/documentation.html">Home</a>
+                <a href="/pages/documentation.html">Docs</a>
+                <a href="https://github.com/rajkishancode/Calm-Designs">Contribute</a>
+            </div>
+        </header>
+
+        <div class="doc-container">
+            <div class="side-bar">
+
+                <div class="nested-list1">
```

Comment:
Suggestion: You can come up with a more descriptive class name here instead of list1. Let me know your thoughts about this?

Reply:
sure i'll do that in every file ,while refactoring the code.